File: dev-docs/snippets/groups/get.swift
import JsBaoClient

// Retrieve a single group by its type and ID.
func get(client: JsBaoClient, groupType: String, groupId: String) async throws {
  // #region example
  let group = try await client.groups.get(groupType: groupType, groupId: groupId)
  // #endregion example
  _ = group
}

File: dev-docs/snippets/groups/get.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Retrieve a single group by its type and ID.
export async function get(
  client: JsBaoClient,
  groupType: string,
  groupId: string,
) {
  // #region example
  const group = await client.groups.get(groupType, groupId);
  // #endregion example
  return group;
}
